### PR TITLE
add link of kubeadm config doc

### LINKED ISSUE #1666: how to get all config like kubectl explain
how to get all config of kubeadm like kubectl explain xxx?
becase `kubeadm config print init-defaults` not print all config any more

## Is this a BUG REPORT or FEATURE REQUEST?

FEATURE REQUEST

## Versions

**kubeadm version** (use `kubeadm version`):
1.15.0
**Environment**:
- **Kubernetes version** (use `kubectl version`): 1.15.0
- **Cloud provider or hardware configuration**:
- **OS** (e.g. from /etc/os-release): centos 7.5
- **Kernel** (e.g. `uname -a`): Linux 192-168-101-55 3.10.0-862.el7.x86_64 #1 SMP Fri Apr 20 16:44:24 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
- **Others**:

## What happened?
`kubeadm config print init-defaults ` or some command print all config of kubeadm

### PR DESCRIPTION
**Pull request by zgfh**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
add link of kubeadm config doc

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubernetes/kubeadm/issues/1666

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/config.go [hunk 1]
```diff
@@ -96,8 +96,10 @@ func NewCmdConfigPrint(out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "print",
 		Short: "Print configuration",
-		Long:  "This command prints configurations for subcommands provided.",
-		RunE:  cmdutil.SubCmdRunE("print"),
+		Long: dedent.Dedent(`
+			This command prints configurations for subcommands provided.
+			For details, see: https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2`),
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/config.go]: this is technically a quick workaround for not having something like `kubectl explain pod.spec` in kubeadm.
/assign @fabriziopandini

**Review by neolit123 — COMMENTED** [review]
thanks for the PR @zgfh

**Review by fabriziopandini — APPROVED** [review]
@zgfh thanks for this change
This can be highly beneficial to users.
/lgtm
/approve

**Pull request merged**